ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -409,6 +410,39 @@ class ProfileMainScreen extends React.PureComponent<Props, State> {
                 dispatchSessionExpired,
                 true
               )}
+            {isDevEnv &&
+              this.debugListItem(
+                I18n.t("profile.main.clearAsyncStorage"),
+                () => {
+                  void AsyncStorage.clear();
+                },
+                true
+              )}
+            {isDevEnv &&
+              this.debugListItem(
+                I18n.t("profile.main.dumpAsyncStorage"),
+                () => {
+                  /* eslint-disable no-console */
+                  console.log("[DUMP START]");
+                  AsyncStorage.getAllKeys()
+                    .then(keys => {
+                      console.log(`\tAvailable keys: ${keys.join(", ")}`);
+                      return Promise.all(
+                        keys.map(key =>
+                          AsyncStorage.getItem(key).then(value => {
+                            // if (/(features)|(euCovidCert)|(mvl)/.test(key)) {
+                            console.log(`\tValue for ${key}\n\t\t`, value);
+                            // }
+                          })
+                        )
+                      );
+                    })
+                    .then(() => console.log("[DUMP END]"))
```

Comment:
```suggestion
                    .then(() => {
                      console.log("[DUMP END]");
                      showToast(
                        "Dump completed. Check logs in your console!",
                        "success"
                      );
                    }
```
Same of previous. It's just 💅🏻 
Skip if you don't like it

Reply:
I see why you are proposing it, though these buttons are here as a hideous reminder that _we didn't have better choices_.
I don't want to encourage this behavior if possible 😃